iothub/device/src/AmqpTransportSettings.cs

```diff
@@ -146,6 +146,12 @@ public TimeSpan OpenTimeout
             set => SetOpenTimeout(value);
         }
 
+        /// <summary>
+        /// A keep-alive for when using web sockets.
```

Comment:
I think it is worth calling out here that this is different from mqtt spec keep-alive. Else we might have people setting this value when using mqtt_ws and then wondering why they aren't receiving a ping response from the service.

Reply:
Please suggest a wording change.

Reply:
A keep-alive for sending ping/pong control frames when using web sockets. This value is different from the protocol level keep-alive packets that are sent over the overlaying MQTT transport protocol.